tests/Admin/AdminTest.php

```diff
@@ -1791,6 +1798,88 @@ public function testGetFilterFieldDescription(): void
         $this->assertSame($bazFieldDescription, $modelAdmin->getFilterFieldDescription('baz'));
     }
 
+    public function testGetDatagrid(): void
+    {
+        $modelManager = $this->createMock(ModelManagerInterface::class);
+        $datagridBuilder = $this->createMock(DatagridBuilderInterface::class);
+        $pager = $this->createMock(PagerInterface::class);
+        $datagrid = $this->createMock(DatagridInterface::class);
+
+        $admin = new CleanAdmin('test code', 'Application\Entity\CleanEntity', 'SonataFooBundle:CleanAdmin');
```

Comment:
Please, use the QCN syntax for controller names.

```suggestion
        $admin = new CleanAdmin('test code', 'Application\Entity\CleanEntity', 'Sonata\FooBundle\CleanAdminController');
```